# 
# Translators:
# Nico Veenstra <nvbgm@gmx.net>, 2023
# Hidde Wieringa <hidde@hiddewieringa.nl>, 2023
# 
msgid ""
msgstr ""
"Project-Id-Version: Lizmap Web Client 3.6 dataviz\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-07-03 14:01++0200\n"
"PO-Revision-Date: 2022-07-29 15:01+0000\n"
"Last-Translator: Hidde Wieringa <hidde@hiddewieringa.nl>, 2023\n"
"Language-Team: Dutch (https://app.transifex.com/3liz-1/teams/62012/nl/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: nl\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: dataviz~dataviz.dock.subtitle
msgctxt "dataviz~dataviz.dock.subtitle"
msgid "Dataviz"
msgstr "Dataviz"

#: dataviz~dataviz.dock.title
msgctxt "dataviz~dataviz.dock.title"
msgid "Dataviz"
msgstr "Dataviz"

#: dataviz~dataviz.log.access_denied.detail
msgctxt "dataviz~dataviz.log.access_denied.detail"
msgid ""
"You have no right to access this ressource. Have you used the right login ?"
msgstr ""
"U heeft geen recht op toegang tot deze bron. Heeft u de juiste login "
"gebruikt?"

#: dataviz~dataviz.log.access_denied.title
msgctxt "dataviz~dataviz.log.access_denied.title"
msgid "Access denied"
msgstr "Geen toegang"

#: dataviz~dataviz.log.invalid_plot_type.detail
msgctxt "dataviz~dataviz.log.invalid_plot_type.detail"
msgid "The given plot type \"%s\" is invalid."
msgstr "Het gegeven plottype \"%s\" is ongeldig."

#: dataviz~dataviz.log.invalid_plot_type.title
msgctxt "dataviz~dataviz.log.invalid_plot_type.title"
msgid "Invalid plot type"
msgstr "Ongeldig plot type"

#: dataviz~dataviz.log.layer_not_found.detail
msgctxt "dataviz~dataviz.log.layer_not_found.detail"
msgid ""
"The layer with the given ID \"%s\" cannot be found in the published QGIS "
"project."
msgstr ""
"De laag met het gegeven ID \"%s\" kan niet worden gevonden in het "
"gepubliceerde QGIS project."

#: dataviz~dataviz.log.layer_not_found.title
msgctxt "dataviz~dataviz.log.layer_not_found.title"
msgid "Layer not found"
msgstr "Laag niet gevonden"

#: dataviz~dataviz.log.no_configuration.title
msgctxt "dataviz~dataviz.log.no_configuration.title"
msgid "There is no dataviz configuration for this project yet."
msgstr "Er is nog geen dataviz configuratie voor dit project."

#: dataviz~dataviz.log.no_data.detail
msgctxt "dataviz~dataviz.log.no_data.detail"
msgid "The plot source data cannot be fetched from the vector layer \"%s\"."
msgstr "De plot gegevensbron kan niet worden opgehaald van de vectorlaag \"%s\"."

#: dataviz~dataviz.log.no_data.title
msgctxt "dataviz~dataviz.log.no_data.title"
msgid "Plot data not available"
msgstr "Plotgegevens niet beschikbaar"

#: dataviz~dataviz.log.no_layers.detail
msgctxt "dataviz~dataviz.log.no_layers.detail"
msgid "The table of layers inside the dataviz configuration is empty."
msgstr "De tabel van lagen in de dataviz configuratie is leeg."

#: dataviz~dataviz.log.no_layers.title
msgctxt "dataviz~dataviz.log.no_layers.title"
msgid "Dataviz layers is empty"
msgstr "Dataviz lagen zijn leeg"

#: dataviz~dataviz.log.plot_configuration_not_found.detail
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.detail"
msgid ""
"The plot configuration cannot be found or parsed by Lizmap dataviz API."
msgstr ""
"De plotconfiguratie kan niet worden gevonden of gelezen door de Lizmap "
"dataviz API."

#: dataviz~dataviz.log.plot_configuration_not_found.title
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.title"
msgid "Plot configuration not found"
msgstr "Plot configuratie niet gevonden"

#: dataviz~dataviz.log.project_not_found.detail
msgctxt "dataviz~dataviz.log.project_not_found.detail"
msgid ""
"The QGIS project \"%s.qgs\" does not exist or is not available in the "
"repository \"%s\"."
msgstr ""
"Het QGIS project \"%s.qgs\" bestaat niet of is niet beschikbaar in de "
"repository \"%s\"."

#: dataviz~dataviz.log.project_not_found.title
msgctxt "dataviz~dataviz.log.project_not_found.title"
msgid "Lizmap project not found"
msgstr "Lizmap project niet gevonden"

#: dataviz~dataviz.log.repository_not_found.detail
msgctxt "dataviz~dataviz.log.repository_not_found.detail"
msgid "The Lizmap repository \"%s\" does not exist or is not available."
msgstr "De Lizmap repository \"%s\" bestaat niet of is niet beschikbaar."

#: dataviz~dataviz.log.repository_not_found.title
msgctxt "dataviz~dataviz.log.repository_not_found.title"
msgid "Lizmap repository not found"
msgstr "Lizmap repository niet gevonden"

#: dataviz~dataviz.log.request_not_supported.detail
msgctxt "dataviz~dataviz.log.request_not_supported.detail"
msgid "The given request \"%s\" is not supported by Lizmap dataviz API."
msgstr "Het verzoek \"%s\" wordt niet ondersteund door de Lizmap dataviz API."

#: dataviz~dataviz.log.request_not_supported.title
msgctxt "dataviz~dataviz.log.request_not_supported.title"
msgid "Request is not supported"
msgstr "Verzoek wordt niet ondersteund"

#: dataviz~dataviz.log.wrong_credentials.detail
msgctxt "dataviz~dataviz.log.wrong_credentials.detail"
msgid "The given login and password are incorrect. Cannot authenticate."
msgstr ""
"De gegeven gebruikersnaam en wachtwoord zijn ongeldig. Can niet inloggen."

#: dataviz~dataviz.log.wrong_credentials.title
msgctxt "dataviz~dataviz.log.wrong_credentials.title"
msgid "Wrong credentials"
msgstr "Verkeerde inloggegevens"
